oaknational/Oak-Web-Application
# feat: key stage navigation component

## Description

- Adds key stage navigation component ( [figma](https://www.figma.com/file/96yUIKNRvg0rMCWasdkJ1u/Design-System-2.0?node-id=745%3A7962) ) to storybook
- Adds key stage nav to teacher home page
- Adds AboutContatBlogList component for duplicate grid section on teacher home and landing page

fixes#[286](https://github.com/oaknational/Samara/issues/244) 

## How to test

1. Go to {storybook deployment}
2. Click on navigation / key stages navigation
3. You should see key-stages nav component


## Checklist
